docs/DeveloperGuide.adoc

```diff
@@ -440,26 +442,140 @@ application, and the following details:
 ** `condition` - Condition of the patient that made the request.
 
 === Filter Feature
-The `filter` command allows users to filter either the current displayed
-list of `HealthWorker` and `Request` objects and replaces the current list
-shown on the UI with the respective filtered lists.
 
-****
-*Notes*:
+With the filter command, users can provide specific parameters that act as
+conditions for HealthHub to filter the `UniqueHealthWorkerList` and
+`UniqueRequestList` by. Using these parameters, HealthHub displays only the
+`HealthWorker` or `Request` whose fields contains the specified parameters.
+
+Command Format: `filter <keyword> <prefix>/<parameter> [<prefix>/<parameter>]
+...`
 
-* The `<prefix>` and `<parameters>` specified in the command descriptions
+[NOTE]
+====
+The `<prefix>` and `<parameters>` specified in the command descriptions
 below follow the same valid prefix and parameter formats as the `add`, `edit`
  and `delete` commands
-****
+====
+
+==== Filtering `HealthWorker`
+
+The command format for filtering `HealthWorker` is:
+
+`filter healthworker/h/1 <prefix>/<parameter> [<prefix>/<parameter>]...`
+
+The fields in `HealthWorker` that the user can filter by are:
+
+* `name` prepended by `n/` prefix
```

Comment:
since the format is already <prefix>/ then should the prefix be without the /?
Then again I think it does not matter much